database/postgres/postgres_team.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict
+from database.models import TeamIndex
+from database.postgres.postgres_country import save_country
+
+def save_team(team_data: Dict, session, sport_id):
```

Comment:
all the same here

we should just pass a Team instance and save it. It's up to business layer to handle all the validation and calculation. We only care about saving to / reading from the database